bin/internal/gradle_wrapper.version

```diff
@@ -0,0 +1 @@
+0b5c1398d1d04ac245a310de98825cb7b3278e2a
```

Comment:
That could trip up parsing. The version files for engine and material fonts do not include such a comment; file name says it all I guess.